testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/x509/X509SingleCertificateBrowserCRLTest.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2024 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.keycloak.testsuite.x509;
+
+import org.jboss.arquillian.drone.api.annotation.Drone;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.keycloak.authentication.authenticators.x509.X509AuthenticatorConfigModel;
+import org.keycloak.authentication.authenticators.x509.X509AuthenticatorConfigModel.IdentityMapperType;
+import org.keycloak.authentication.authenticators.x509.X509AuthenticatorConfigModel.MappingSourceType;
+import org.keycloak.testsuite.util.PhantomJSBrowser;
+import org.openqa.selenium.WebDriver;
+
+/**
+ *
+ * @author rmartinc
+ */
+public class X509SingleCertificateBrowserCRLTest extends AbstractX509AuthenticationTest {
+
+    @ClassRule
+    public static CRLRule crlRule = new CRLRule();
+
+    @Drone
+    @PhantomJSBrowser
+    private WebDriver phantomJS;
```

Comment:
Ideally we'd try to avoid introducing more PhantomJS (see #9979), but since `AbstractX509AuthenticationTest` is heavily based on it there is very little we can do.

Reply:
All x509 tests use the phantom, so I just followed the same idea. So when the x509 tests were changed to use another thing this one will be just one more test to change. Not doing anything here.

Reply:
Indeed, nothing that can be done about it until the testing team gets to it :)

Reply:
+1 , FYI. There is WiP by @Aboullos from QA team for refactoring of X.509 tests to make them get rid of PhantomJS